api/src/glfs-fops.c

```diff
@@ -5401,8 +5390,6 @@ pub_glfs_fd_set_lkowner(struct glfs_fd *glfd, void *data, int len)
         goto invalid_fs;
     }
 
-    GF_VALIDATE_OR_GOTO(this->name, data, out);
```

Comment:
This is a public function. We don't control the callers, so argument check is a must.

Reply:
done